docker/utils/utils.py

```diff
@@ -498,6 +500,15 @@ def create_host_config(
             memswap_limit = parse_bytes(memswap_limit)
         host_config['MemorySwap'] = memswap_limit
 
+    if mem_swappiness is not None:
+        if version_lt(version, '1.20'):
+            raise errors.InvalidVersion(
+                'mem_swappiness param not supported for API version < 1.20'
+            )
+        if isinstance(mem_swappiness, six.string_types):
+            mem_swappiness = parse_bytes(mem_swappiness)
```

Comment:
Why `parse_bytes` here? Isn't it supposed to be a simple integer value between 0 and 100?